### PR TITLE
[DAT-486] - Checkout process - Informacion de contacto - Integrar los estilos de la librería de UI de Doppler

### PR DESCRIPTION
**Pull request by jhoffman-ms**
Apply the new styles for the 'Checkout' page and 'Contact Information' section.

**'Contact Information' section:**

**Before:**

image

**After:**

image

'Checkout' page:

Before:

image

After:

image

### CODE REVIEW

File: src/components/ChangePlan/Checkout/Checkout.js [hunk 1]
```diff
@@ -57,29 +57,42 @@ const Checkout = () => {
...
-              complete={false}
-            ></Step>
+            <div className="dp-wrapper-payment-process">
+              <ul className="dp-accordion">
+                <Step
+                  active={activeStep === checkoutSteps.contactInformation}
+                  title={_('checkoutProcessForm.contact_information_title')}
+                  complete={complete}
+                  stepNumber={1}
+                  onActivate={() => {
```
Comment (jcamposmk) [inline src/components/ChangePlan/Checkout/Checkout.js:71]:
```suggestion
onActivate={() => setActiveStep(checkoutSteps.contactInformation)}
```

Comment (jhoffman-ms) [reply src/components/ChangePlan/Checkout/Checkout.js]: Done

**Comment by jhoffman-ms** [comment]
Do you have another feedback or is ready to merge?.

**Comment by jcamposmk** [comment]
do you think we can meet tomorrow so you can tell me about this? and do a more exhaustive review @jhoffman-ms

**Comment by jcamposmk** [comment]
:+1:

File: src/components/ChangePlan/Checkout/Step/Step.js [hunk 2]
```diff
@@ -2,48 +2,46 @@ import React from 'react';
...
-  marginTop: '20px',
-};
-
 const displaFlexAndjustifyContentSpaceBetweenStyle = {
   display: 'flex',
   justifyContent: 'space-between',
 };
 
-const upperCaseStyle = {
+const titleStyle = {
```
Comment (cbernat) [inline src/components/ChangePlan/Checkout/Step/Step.js:10]: @jhoffman-ms Why do we still have inline styles here? Now with the new styles applied this should not be here, or be made into a styled component.

Comment (jhoffman-ms) [reply src/components/ChangePlan/Checkout/Step/Step.js:10]: Because Gus didn't work with this part yet. For it reason I maintain the inline style.

Comment (cbernat) [reply src/components/ChangePlan/Checkout/Step/Step.js:10]: Perhaps add a TODO comment later then.

Comment (jhoffman-ms) [reply src/components/ChangePlan/Checkout/Step/Step.js:10]: Currently in the file I added the following comment:

``` 
/* TODO: These styles will be removed with the correct classes. Currently Gus is working with the layout. */
```

**Pull request merged**